src/message/messageActionSheet.js

```diff
@@ -436,12 +413,22 @@ export const showTopicActionSheet = ({
     unreadStreams: UnreadStreamsState,
     ...
   }>,
-  stream: string,
+  streamId: number,
   topic: string,
 |}): void => {
+  const sub = backgroundData.subscriptions.find(x => x.stream_id === streamId);
+  try {
+    if (!sub) {
+      throw Error('Stream id does not exist.');
+    }
+  } catch (err) {
+    logging.error(err);
+    return;
+  }
```

Comment:
I don't think there's a reason to use `try`/`catch` here, we can just do:

```
if (!sub) {
  logging.error('Stream id does not exist.');
  return;
}
```

instead.